src/script/lua_api/l_server.cpp

```diff
@@ -233,11 +233,11 @@ int ModApiServer::l_get_player_information(lua_State *L)
 	lua_pushnumber(L, prot_vers);
 	lua_settable(L, table);
 
-#ifndef NDEBUG
 	lua_pushstring(L,"serialization_version");
 	lua_pushnumber(L, ser_vers);
 	lua_settable(L, table);
 
+#ifndef NDEBUG
```

Comment:
It seems to me that `serialization_version` would now be included even in Release builds. Why have you moved `#ifndef NDEBUG` further down?

Reply:
It was decided to allow ser_vers

https://github.com/minetest/minetest/pull/8616#issuecomment-515211734

https://github.com/minetest/minetest/pull/8616#issuecomment-520265370

Reply:
Ah ok, thanks for clarifying